api/src/opentrons/protocol_api/core/engine/instrument.py

```diff
@@ -65,6 +65,25 @@ def aspirate(
         rate: float,
         flow_rate: float,
     ) -> None:
+        """Aspirate a given volume of liquid from the specified location.
+
+        :param volume: The volume of liquid to aspirate, in microliters.
+        :type volume: int or float
+
+        :param location: Where to aspirate from. The
+                         robot will aspirate from the exact specified location.
+
+        :param well_core: Protocol engine based well core implementation.
+        :type well_core: optional WellCore
+
+        :param rate: Not used in engine based aspirate.
+                        Part of the abstract class for legacy purposes.
+        :type rate: float
+
+        :param flow_rate: The absolute aspirate flow rate. calculated by:
+                        `rate` * :py:attr:`default_flow_rate.aspirate`.
+        :type rate: float
+        """
```

Comment:
Please use Google-style doc strings in internal classes, per our flake8 config.

Eventually we'll be able to move the docstrings of the public classes over to this style, too.

```suggestion
        """Aspirate a given volume of liquid from the specified location.

        Args:
            volume: The volume of liquid to aspirate, in microliters.
            location: The exact location to aspirate from.
            well_core: The well to aspirate from, if applicable.
            rate: Not used in this core.
            flow_rate: The flow rate in µL/s to aspirate at.
        """
```